### PR TITLE
WIP: dynamic args

### PR DESCRIPTION
**Pull request by Alexays**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
       before_install:
         - sudo sed -i '' 's/quarterly/latest/' /etc/pkg/FreeBSD.conf
         - sudo pkg install -y date gtk-layer-shell gtkmm30 jsoncpp libdbusmenu
-               libfmt libmpdclient libudev-devd meson pulseaudio scdoc spdlog
+               libfmt libmpdclient libudev-devd meson pulseaudio scdoc spdlog libinput libxml2
```
Comment (jbeich) [inline .travis.yml:34]: - If you plan to use libinput (still no refereces in the code) maybe revert 74db69dcb7cc as well
- libxml2 (no refereces in the code) is already installed as a dependency of **many** packages:
```
$ pkg info -r libxml2
libxml2-2.9.10:
        libdbusmenu-16.04.0_2
        gtkmm30-3.22.2
        consolekit2-1.2.1_1
        pangomm-2.40.1_4
        libxslt-1.1.34_1
        cairomm-1.12.2_4
        librsvg2-2.40.21
        libxml++-2.40.1,1
        libgsf-1.14.46
        libcroco-0.6.13
        gdk-pixbuf2-2.40.0
        shared-mime-info-1.10_2
        xkeyboard-config-2.29
        glib-2.56.3_7,1
        libdrm-2.4.100,1
        wayland-1.18.0_4
        libxcb-1.13.1
```

Comment (Alexays) [reply .travis.yml:34]: Will be used by #659

Comment (jbeich) [reply .travis.yml:34]: I've looked at #659 and couldn't find libinput or libxml2 references there. Do you mean that PR is also work-in-progress?

**Comment by betaboon** [comment]
I'm still very much interested in this feature. what's the state of it? can we help ?

**Comment by Anakael** [comment]
Can you provide some description for such changes and it shoud be used?

**Comment by Alexays** [comment]
> Can you provide some description for such changes and it shoud be used?

The goal was to avoid processing some args that will not be used later.

**Pull request closed without merging**